up42/job.py

```diff
@@ -3,33 +3,21 @@
 from time import sleep
 from typing import Dict, List, Union, Optional
 
-import folium
 from geopandas import GeoDataFrame
 import geopandas as gpd
-import numpy as np
 import requests
 import requests.exceptions
-from shapely.geometry import box
-from rasterio.vrt import WarpedVRT
-import rasterio
 
 from up42.auth import Auth
 from up42.jobtask import JobTask
 from up42.tools import Tools
 from up42.utils import (
     get_logger,
-    folium_base_map,
+    _map_images,
     download_results_from_gcs,
     download_results_from_gcs_without_unpacking,
 )
 
-try:
-    from IPython.display import display
-    from IPython import get_ipython
-except ImportError:
-    # No Ipython installed, Installed but run in shell
-    pass
-
```

Comment:
I guess this is ok to remove because we don't use the display any more correct?

Reply:
Right, for your info @chrieke